Content.Server/Cargo/Systems/CargoSystem.Shuttle.cs

```diff
@@ -190,7 +194,17 @@ private List<(EntityUid Entity, CargoPalletComponent Component, TransformCompone
                 continue;
             }
 
+            if (requestType == BuySellType.sell && comp.PalletType != BuySellType.sell && comp.PalletType != BuySellType.both)
+            {
+                continue;
+            }
+            if (requestType == BuySellType.buy && comp.PalletType != BuySellType.buy && comp.PalletType != BuySellType.both)
+            {
+                continue;
+            }
+
```

Comment:
Just use bitmasks for this (see stuff like (X & Y) != 0x0)

Reply:
I have no idea how to do any of this stuff, my dude

Reply:
You're gonna have to elaborate on this one.